### ISSUE TITLE
ARC data Controller creation fails due to space in Azure location name

### ISSUE THREAD

**Issue by Nagi-MSFT** [microsoft/azure_arc#165]
**Describe the bug**
While using the azure arc data jumpstart arm template, it creates the aks cluster and then the jumpbox vm. however when you login to the jumpbox vm, the logon script fails at data controller creation as the environment variable ARC_DC_REGION contains a space and it doesnt pickup the complete name. When i was trying, i chose southeast asia and because of the space, the controller creation fails.

**To Reproduce**
1. choose southeast asia as region for the deployment in the parameter file
2. proceed with normal deployment
3. after vm creation login to the jumpbox, and logon script should start 
4. after a while the script errors out saying invalid region name.

**Expected behavior**
data controller should have been created irrespective of the selected region, may be the space between the region name should be eliminated by some replace function before setting the environment variable

**Comment by likamrat** [comment]
In your ARM template parameters file, try "southeastasia" instead "southeast asia". You should be using the region id and not the region display name.

**Comment by likamrat** [comment]
@Nagi-MSFT any updates?

**Comment by Nagi-MSFT** [comment]
I haven’t retried it. I just modified the environment variable after it failed to kick start again and it went through. However found another issue where the data studio was not connected to arc due to extensions not being installed correctly. And the VM has security control which doesn’t allow to download from internet. I then got it connected on my laptop by exposing firewalls.
I will retry the region id and comeback if it solves.

Thanks

[cid:image001.png@01D6CD48.55539A10]<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fazure.microsoft.com%2F&data=02%7C01%7Cnavirdi%40microsoft.com%7C241af741467b4364272a08d83e681723%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637327964088030858&sdata=G5LlJVPRfFKEjSRD4LDn74dPSUXv7plL7Opyzuv94ro%3D&reserved=0>Naginder Singh Virdi
Global Blackbelt Technical Specialist – Open Source Databases | Microsoft
Tel +65 68887202(Office), +65 83019663(mobile)

**Comment by Nagi-MSFT** [comment]
I haven’t retried it. I just modified the environment variable after it failed to kick start again and it went through. However found another issue where the data studio was not connected to arc due to extensions not being installed correctly. And the VM has security control which doesn’t allow to download from internet. I then got it connected on my laptop by exposing firewalls.
I will retry the region id and comeback if it solves.

Thanks
